src/daemon/daemon.cpp

```diff
@@ -3090,12 +3090,20 @@ mp::Daemon::async_wait_for_ready_all(grpc::ServerReaderWriterInterface<Reply, Re
                     if (!warned_exec_failure)
                     {
                         add_fmt_to(warnings,
-                                   "failure configuring network interfaces in {}, you can still do it manually.\n",
+                                   "Failure configuring network interfaces in {}: is Netplan installed?\n"
+                                   "You can still configure them manually.\n",
                                    name);
                     }
 
                     warned_exec_failure = true;
                 }
+                catch (const SSHException&) // The SSH session could not be created.
+                {
+                    add_fmt_to(warnings,
+                               "Cannot create a SSH shell to execute commands on {}, you can configure new\n"
+                               "interfaces manually Netplan once logged to the instance.\n",
```

Comment:
Maybe "...manually **via** Netplan... "

Reply:
Done! I also added some test for these newly added lines.